### PR TITLE
PIL-1242-Fiat formatting change

### PR DESCRIPTION
**Pull request by vignesha22**
image
just a sample image of how it will looks if the value is less than 0.01 but its not on code as from the pic above the token fiat value is 0 (it will display as zero).

### CODE REVIEW

File: src/utils/format.js [hunk 1]
```diff
@@ -130,7 +130,11 @@ export function formatFiatValue(value: ?BigNumber | number, currency?: string, o
   const formattedValue = formatValue(value, { decimalPlaces: 2, ...options });
   if (!formattedValue) return null;
 
-  return currency ? t('fiatValue', { value: formattedValue, symbol: getCurrencySymbol(currency) }) : formattedValue;
+  if (!currency) return formattedValue;
+
+  const currencyValue = t('fiatValue', { value: formattedValue, symbol: getCurrencySymbol(currency) });
+  // eslint-disable-next-line max-len
+  return BigNumber(formattedValue).lt(0.01) && BigNumber(formattedValue).gt(0) ? ('< ').concat(currencyValue) : currencyValue;
```
Comment (IAmKio) [inline src/utils/format.js:137]: Split this into multiple lines please to make it cleaner, for example:

```suggestion
  return BigNumber(formattedValue)
    .lt(0.01) && BigNumber(formattedValue).gt(0) ? ('< ').concat(currencyValue) : currencyValue;
```

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a minor change

File: src/utils/format.js [hunk 2]
```diff
@@ -130,7 +130,13 @@ export function formatFiatValue(value: ?BigNumber | number, currency?: string, o
   const formattedValue = formatValue(value, { decimalPlaces: 2, ...options });
   if (!formattedValue) return null;
 
-  return currency ? t('fiatValue', { value: formattedValue, symbol: getCurrencySymbol(currency) }) : formattedValue;
+  if (!currency) return formattedValue;
+
+  const currencyValue = t('fiatValue', { value: formattedValue, symbol: getCurrencySymbol(currency) });
+  // eslint-disable-next-line max-len
```
Comment (divyaBacancy) [inline src/utils/format.js:136]: Remove this line as it is not required now

**Review by divyaBacancy — APPROVED** [review]
Just a minor change, otherwise LGTM

**Review by IAmKio — APPROVED** [review]

**Pull request merged**